src/test/java/com/training/services/tests/LoyalityUpdateTest.java

```diff
@@ -74,4 +75,99 @@ public void testLoyality() {
         softAssert.assertThat(loyaltyResponse.getPayload().getRelationshipPoints()).isEqualTo("83")
                 .as("Relationship points is not equal to 83");
     }
+
+    /**
+     * Putting wrong app key, In loyality API, and checking if we are getting correct response from sever
+     */
+    @Test
+    public void testLoyalityGANegativeWrongAppKey() {
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyWrongValue"));
+
+        com.training.services.ga.loyalty.Response loyaltyNegativeResponse
+                = new RestEngine().getResponsePut(map.get("url_base") + "/v1/guestAccounts/loyalty",
```

Comment:
can we create a global variable url = map.get("url_base") + "/v1/guestAccounts/loyalty"
and use it all our tests?